named_entity_extractor.py

```diff
@@ -0,0 +1,112 @@
+""" Wrapper for all GCP Language API code.
+
+You can read more about it here: https://cloud.google.com/natural-language/docs/basics
+You will have to export GOOGLE_APPLICATION_CREDENTIALS before running this file.
+export GOOGLE_APPLICATION_CREDENTIALS=[PATH TO credentials.json]
+
+You can generate a new key here: https://console.cloud.google.com/apis/credentials/serviceaccountkey?project=nyupoladstransparency&folder=&organizationId=&angularJsUrl=%2Fapis%2Fcredentials%2Fserviceaccountkey%3Fsupportedpurview%3Dproject%26project%3Dnyupoladstransparency%26folder%3D%26organizationId%3D&supportedpurview=project
+"""
+import logging
+from collections import defaultdict
+
+from google.cloud import language_v1
+from google.cloud.language_v1 import enums
+from google.protobuf.json_format import MessageToDict
+
+GCS_CREDENTIALS_FILE = '/home/divam/credentials.json'
```

Comment:
might want to remove, or change to just 'credentials.json'

Reply:
Done.